superset/connectors/sqla/models.py

```diff
@@ -61,10 +61,18 @@
 config = app.config
 metadata = Model.metadata  # pylint: disable=no-member
 
-SqlaQuery = namedtuple(
-    "SqlaQuery", ["sqla_query", "labels_expected", "extra_cache_keys"]
-)
-QueryStringExtended = namedtuple("QueryStringExtended", ["sql", "labels_expected"])
+
+class SqlaQuery(NamedTuple):
```

Comment:
Oh wow, the new `NamedTuple` are so much better, always hated the way the old ones were delcared...